content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -16,22 +15,65 @@ options(
   install.packages.check.source = "no",
   install.packages.compile.from.source = "never"
 )
-if (!"checklist" %in% rownames(utils::installed.packages())) {
-  utils::install.packages("checklist")
+if (
+  !all(
+    c("checklist", "fortunes", "remotes") %in%
+    rownames(utils::installed.packages())
+  )
+) {
+  if (Sys.getenv("RSTUDIO") == "1") {
+    warning(
+      "Missing packages. ",
+      "Please close RStudio and run R-",
+      paste(R.Version()[c("major", "minor")], collapse = "."),
+      "-win.exe first to install them automatically."
+    )
+  } else {
+    # remove any existing LOCK file
+    file.remove(
+      list.files(
+        file.path(.libPaths(), "00LOCK"), recursive = TRUE,
+        full.names = TRUE
+      )
+    )
```

Comment:
Perhaps  a safer alternative is to error with a message, asking whether another installation process is running. And if not, either request the user to do this manually, or based on user interaction with y/n.

Reply:
fixed in c9948f0